beta/src/components/Search.tsx

```diff
@@ -108,10 +119,12 @@ export const Search: React.FC<SearchProps> = ({
         onClick={onOpen}>
         <IconSearch className="mr-3 align-middle text-gray-30 flex-shrink-0 group-betterhover:hover:text-gray-70" />
         Search
-        <span className="ml-auto hidden sm:flex item-center">
-          <Kbd>⌘</Kbd>
-          <Kbd>K</Kbd>
-        </span>
+        {showKeyboardShortcuts && (
+          <span className="ml-auto hidden sm:flex item-center">
```

Comment:
By default it is hidden with CSS. `sm:flex` for devices other than small ones only we display the shortcuts. `showKeyboardShortcuts` we might not need this ?

Reply:
I'm using `showKeyboardShortcuts` as a way to prevent "flashing". On a Mac-like device with the current code, without this check, the shortcut key would first show "Ctrl" and then go to the command key after the browser has been checked (`showMacosShortcut` defaults to false). 

It is acting as conditional that represents if the user-agent has been checked or not.